package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cfn-modules/fargate-service",
-  "version": "1.1.1",
+  "version": "1.2.1",
```

Comment:
should be `1.2.0`. Please also update the CloudFormation output value to `1.2.0`